docs/platform/using-airbyte/workspaces.md

```diff
@@ -2,63 +2,86 @@
 products: cloud, oss-enterprise
 ---
 
-# Manage your workspace
+# Manage workspaces
 
-A workspace in Airbyte allows you to collaborate with other users and manage connections together.
+You can create, manage, and delete workspaces, and control access to them.
 
-## Add users to your workspace
+## Create a new workspace
```

Comment:
Might be worth noting that you need an Organization Editor or Organization Admin role to do this.